README.md

```
@@ -18,6 +18,9 @@ TensorRT LLM
 <div align="left">
 
 ## Tech Blogs
+* [09/26] Inference Time Compute Implementation in TensorRT LLM
+✨ [➡️ link](./docs/source/blogs/tech_blog/blog13_Inference_Time_Compute_Implementation_in_TensorRT-LLM.md)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)